examples/ws2812_i2s/ws2812_i2s_colour_loop.c

```diff
@@ -69,15 +71,23 @@ static void demo(void *pvParameters)
             memset(&pixels[fix_index(head_index - tail_length)], 0, 
                     sizeof(ws2812_pixel_t));
 
-            ws2812_i2s_update(pixels);
-            vTaskDelay(20 / portTICK_PERIOD_MS);
+            ws2812_i2s_update(pixels, PIXEL_RGBW);
+            vTaskDelay(50 / portTICK_PERIOD_MS);
         }
     }
 }
 
 void user_init(void)
 {
     uart_set_baud(0, 115200);
+    struct sdk_station_config config = {
+        .ssid = "Loading...",
+        .password = "morays59924_howitzer",
+    };
+
+    /* required to call wifi_set_opmode before station_set_config */
+    sdk_wifi_set_opmode(STATION_MODE);
+    sdk_wifi_station_set_config(&config);
```

Comment:
I don't think that WiFi-related things are necessary here

Reply:
Oops, that is not meant to be there...